tests/integration_tests/test_integration_dds.py

```diff
@@ -298,10 +363,7 @@ def test_can_deliver_and_respond(self):
                         'auth_token': '1234',
                         'skip_mover': False,
                         }
-                delivery_response = self.http_client.fetch(self.get_url(delivery_url), method='POST', body=json.dumps(delivery_body))
-
-                staging_response = yield self.http_client.fetch(staging_status_links["ABC_123"])
-                self.assertEqual(json.loads(staging_response.body)["status"], StagingStatus.staging_successful.name)
+                delivery_response = yield self.http_client.fetch(self.get_url(delivery_url), method='POST', body=json.dumps(delivery_body))
```

Comment:
Should we check that the response is OK? Also, I don't really get the point of this test if we don't wait to confirm that the long delivery finishes ok? But maybe I'm missing something.

Reply:
This test times out if the service waits for the delivery to complete before responding. But yeah, maybe I should add some comments so that we remember what this test is for in the future.